jujugui/static/gui/src/app/components/store/_store.scss

```diff
@@ -185,6 +185,20 @@
       left: 0;
       right: 0;
     }
+
+    &__block-link {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      display: block;
+    }
+
+    .button--inline-neutral {
+      z-index: 1;
```

Comment:
There is a z-index fn that should probably be used to keep things in the right order.